apps/meteor/server/models/raw/EmailInbox.ts

```diff
@@ -12,4 +12,32 @@ export class EmailInboxRaw extends BaseRaw<IEmailInbox> implements IEmailInboxMo
 	protected modelIndexes(): IndexDescription[] {
 		return [{ key: { email: 1 }, unique: true }];
 	}
+
+	async setDisabledById(id: IEmailInbox['_id']): Promise<ModifyResult<IEmailInbox>> {
+		return this.findOneAndUpdate({ _id: id, active: true }, { $set: { active: false } }, { returnDocument: 'after' });
+	}
+
+	async removeById(id: IEmailInbox['_id']): Promise<DeleteResult> {
```

Comment:
`removeById` already exists on `BaseRaw`.. you can just remove this one..

https://github.com/RocketChat/Rocket.Chat/blob/de1960f4a0ce21e13919991023c88054afced4fd/apps/meteor/server/models/raw/BaseRaw.ts#L266